aws/structure.go

```diff
@@ -2834,8 +2834,6 @@ func flattenIoTRuleDynamoDbActions(actions []*iot.Action) []map[string]interface
 		if v != nil {
 			result["hash_key_field"] = *v.HashKeyField
 			result["hash_key_value"] = *v.HashKeyValue
-			result["range_key_field"] = *v.RangeKeyField
```

Comment:
Nit: As a good practice for all these with `*` dereferences around here -- we can save ourselves the trouble of `nil` checking these (and prevent potential panics) using the AWS Go SDK provided helper `aws.StringValue(v.XXX)`

Reply:
:+1: